ui/js/graphs.js

```diff
@@ -396,6 +396,8 @@ perf.controller('GraphsCtrl', [
             $scope.selectedDataPoint = getSeriesDataPoint(item);
             showTooltip($scope.selectedDataPoint);
             updateSelectedItem();
+            if ($.inArray($scope.tooltipContent.revision, $scope.highlightedRevisions) < 0) $scope.highlightedRevisions.push($scope.tooltipContent.revision);
```

Comment:
Thanks @akhileshpillai!

I think this `if` is missing it's `{` block statement?

The line is also 157 characters long. We'd like to wrap it at ~80-100 or so.